apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+You can decide whether you want to automatically stop, or downscale, instances of your app when they're idle and automatically restart them when they're needed. Each instance of your app is running on a Fly machine. Fly Proxy controls automatically starting and automaticaly stopping machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
```

Comment:
We need to mention that this applies only to V2 apps.  Autoscaling in Apps V1 is Nomad-controlled. Also doesn't really work but see https://fly.io/docs/apps/legacy-scaling/ for scaling of V1 apps.

(also whoops https://fly.io/docs/flyctl/autoscale/ we need to tell flyctl to say that's only for V1 apps) 

It'll be confusing to call this downscaling. It stops a running machine but leaves it in place. We're not calling this autoscaling in general. It only works within the set of machines you’ve already provisioned. `fly scale count` (which now exists for V2: https://github.com/superfly/flyctl/pull/1983) creates or destroys Machines.

Note that we don't call Machines instances. It's too Nomady.

(typo automaticaly)

Reply:
Another point: this is a feature of Fly Proxy, so only works on Machines with services configured. (You can still get a Machine with or without services to stop by having its main process exit)

Reply:
@catflydotio 
So, do we remove the banner then as well?

I got "downscaling" from some logs in this community post: https://community.fly.io/t/what-does-downscaling-app-mean/12309. I wanted to connect the terms somehow. Thoughts?

Reply:
> So, do we remove the banner then as well?

Since nothing in this doc applies to Nomad apps, we could have a banner or callout saying "this applies to V2 apps only; see xyz doc for scaling V1 apps" instead of the "rolling out v2 banner" (which itself could use an update)

> Are you saying replace all instances of "instance" with "machine" in this doc?

Where the sentence still makes sense, yes.

Reply:
> I got "downscaling" from some logs in this community post: https://community.fly.io/t/what-does-downscaling-app-mean/12309. I wanted to connect the terms somehow. Thoughts?

My thought on this is that the logs shouldn't call it downscaling. :)